frontend/src/components/Modal/Modal.js

```diff
@@ -0,0 +1,49 @@
+import React, { useState, useRef } from "react";
+import "./Modal.scss";
+import Button from "../Button/Button";
+
+const Modal = ({ header, body, footer }) => {
+  const [modalShow, setModalShow] = useState(true);
+  const modalRef = useRef();
+  const handleClick = () => {
+    setModalShow(false);
+  }
+
+  window.addEventListener("click", function (e) {
+    if (e.target === modalRef.current) {
+      setModalShow(false);
+    }
+  });
+
+   header="Sign Up!"
+   body = "Recieve updates and stay informed!"
+   footer="*this is a disclaimer"
```

Comment:
Don't set props in the component. They have to be passed like `<Modal header="Sign up" />` But it's fine if you just remove them, since we want it to be reusable.